app/src/comic/views.py

```
@@ -0,0 +1,141 @@
+            )
+
+        if User.objects.filter(username=username).exists():
+            return Response(
+                {"error": "用戶名已存在"}, status=status.HTTP_400_BAD_REQUEST
+            )
+
+        try:
+            user = User.objects.create_user(
+                username=username, password=password, email=email
+            )
+        except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)